setup.cfg

```diff
@@ -228,10 +228,12 @@ pretty = False
 # Warns about per-module sections in the config file that do not match any files processed when invoking mypy.
 warn_unused_configs = True
 # Make arguments prepended via Concatenate be truly positional-only.
-strict_concatenate = True
+extra_checks = True
 # Allows variables to be redefined with an arbitrary type,
 # as long as the redefinition is in the same block and nesting level as the original definition.
 # allow_redefinition = True
+# ForkServerContext is not supported on Windows.
+platform = linux
```

Comment:
Perhaps ignore this check instead of hard coding to Linux?
```
monai\apps\auto3dseg\data_analyzer.py:15:1: error: Module "multiprocessing.context" has no attribute "ForkServerContext"  [attr-defined]
```

Reply:
I don't think this will work since on Linux this ignore would be redundant and therefore would then fail the pipeline. I know, that hardcoding it to Linux is also not optimal, but at least we have a reproducable check on all platforms. BTW it doesn't have to be Linux, it just needs to be anything else than Win32, if you prefer any other platform. https://docs.python.org/3/library/sys.html#sys.platform

Reply:
I don't see where this error is coming from given the code line that's referenced or anywhere in code the context is referred to. I think not running mypy on win32 would be a problem for those developing on Windows. CC @wyli @Nic-Ma

Reply:
@ericspod I've updated the PR to type `tmp_ctx` as `Any`. I think this is the best solution.